common/src/main/java/org/jboss/hal/testsuite/page/runtime/UndertowRuntimePage.java

```diff
@@ -9,7 +9,7 @@
 @Place(NameTokens.UNDERTOW_RUNTIME_DEPLOYMENT_VIEW)
 public class UndertowRuntimePage extends BasePage {
 
-    @FindBy(id = "undertow-deployment-session_wrapper")
+    @FindBy(id = "undertow-org.jboss.hal.testsuite.test.deployment-session_wrapper")
```

Comment:
Can you please share some details about this chage? Why this needs to be changed?

Reply:
Thanks for spotting this! This actually looks like a refactoring error and most probably is, will revert this.